core/common/src/test/java/alluxio/PropertyKeyTest.java

```diff
@@ -58,6 +72,12 @@ public void isValid() throws Exception {
   }
 
   @Test
+  public void aliasIsValid() throws Exception {
+    assertTrue(PropertyKey.isValid(mTestProperty.toString()));
+    assertTrue(PropertyKey.isValid(mNullAliasTestProperty.toString()));
```

Comment:
I am sorry, I was intended to test if the isValid method will work for the alias, and check if null will not be as a valid key, but I forgot to change the code. Here is the corresponding code:
`assertTrue(PropertyKey.isValid("alluxio.test.property.alias1"));
  assertTrue(PropertyKey.isValid("alluxio.test.property.alias2"));
  assertFalse(PropertyKey.isValid(null));`
Is that ok?